src/pages/saksbehandling/stans/Stans.tsx

```diff
@@ -0,0 +1,209 @@
+import * as RemoteData from '@devexperts/remote-data-ts';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Knapp } from 'nav-frontend-knapper';
+import { Label, Select, Textarea } from 'nav-frontend-skjema';
+import { Feilmelding, Innholdstittel } from 'nav-frontend-typografi';
+import React from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { Route, Switch, useHistory } from 'react-router-dom';
+
+import ApiErrorAlert from '~components/apiErrorAlert/ApiErrorAlert';
+import DatePicker from '~components/datePicker/DatePicker';
+import * as revurderingActions from '~features/revurdering/revurderingActions';
+import sharedMessages from '~features/revurdering/sharedMessages-nb';
+import { useAsyncActionCreator } from '~lib/hooks';
+import { useI18n } from '~lib/i18n';
+import * as Routes from '~lib/routes';
+import { Nullable } from '~lib/types';
+import yup from '~lib/validering';
+import { OpprettetRevurderingGrunn, Revurdering } from '~types/Revurdering';
+import { Sak } from '~types/Sak';
+import { StansAvYtelse } from '~types/Stans';
+import { getRevurderingsårsakMessageId } from '~utils/revurdering/revurderingUtils';
+
+import messages from './stans-nb';
+import styles from './stans.module.less';
+import StansOppsummering from './stansOppsummering';
+
+interface Props {
+    sak: Sak;
+}
+
+interface FormData {
+    årsak: Nullable<OpprettetRevurderingGrunn>;
+    stansDato: Nullable<Date>;
+    begrunnelse: string;
+}
+
+function hentDefaultVerdier(r: Nullable<Revurdering>): FormData {
+    if (r) {
+        return {
+            stansDato: new Date(r.periode.fraOgMed),
+            begrunnelse: r.begrunnelse ?? '',
+            årsak: r.årsak,
+        };
+    }
+
+    return {
+        stansDato: null,
+        begrunnelse: '',
+        årsak: null,
+    };
+}
+
+const Stans = (props: Props) => {
+    const history = useHistory();
+    const urlParams = Routes.useRouteParams<typeof Routes.stansRoute>();
+    const revurdering = props.sak.revurderinger.find((r) => r.id === urlParams.revurderingId) ?? null;
+
+    const { intl } = useI18n({ messages: { ...messages, ...sharedMessages } });
+
+    const [opprettStatus, opprettStans] = useAsyncActionCreator(revurderingActions.opprettStans);
+    const [oppdaterStatus, oppdaterStans] = useAsyncActionCreator(revurderingActions.oppdaterStans);
+    const status = RemoteData.combine(opprettStatus, oppdaterStatus);
+
+    const { ...form } = useForm<FormData>({
```

Comment:
Hvis du ikke trenger noen destrukturering så kan du gjøre
```suggestion
    const form = useForm<FormData>({
```

Men du burde nok vise en feiloppsummering under skjemaet.